release/kata-deploy-binaries.sh

```diff
@@ -115,10 +115,10 @@ install_qemu() {
 # Install static firecracker asset
 install_firecracker() {
 	info "build static firecracker"
-	"${script_dir}/../static-build/firecracker/build-static-firecracker.sh"
+	[ -d firecracker ] || "${script_dir}/../static-build/firecracker/build-static-firecracker.sh"
```

Comment:
Indented one extra tab. Also suggesting to use `[ -f firecracker/firecraker-static ] || ...`

Reply:
About  indentation, I was referring to L120-121, sorry.

Reply:
done